Formula/awscli.rb

```diff
@@ -1,8 +1,10 @@
 class Awscli < Formula
   desc "Official Amazon AWS command-line interface"
   homepage "https://aws.amazon.com/cli/"
+  # Make sure that you also bump botocore
```

Comment:
Put this above the `class Awscli < Formula` line, or above the `botocore` resource line.